sql/dd/impl/types/object_table_impl.h

```diff
@@ -35,7 +35,7 @@ namespace dd {
 class Object_table_impl : virtual public Object_table {
  protected:
   mutable uint m_last_dd_version;
-  Object_table_definition_impl m_target_def;
+  mutable Object_table_definition_impl m_target_def;
```

Comment:
Because we want to remove "encryption" option only for the time of updating dd_properties. Once System_tables container is filled with system tables they are only available as cont objects. We create system tables with encryption='y' to satisfy the requirement for encrypted mysql tablespace.